packages/datagateway-dataview/src/page/translatedHomePage.component.tsx

```diff
@@ -30,7 +30,9 @@ export const TranslatedHomePage = React.memo(
         decal2Image={props.pluginHost + Decal2Image}
         decal2DarkImage={props.pluginHost + Decal2DarkImage}
         decal2DarkHCImage={props.pluginHost + Decal2DarkHCImage}
-        facilityImage={props.pluginHost + FacilityImage}
+        facilityImage={
+          props.facilityImageURL ? props.facilityImageURL : FacilityImage
```

Comment:
I think the last part of this should be `(props.pluginHost + FacilityImage)`

Reply:
Yes, thanks for pointing that out, I've fixed it 👍